.github/workflows/pd-tests.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
         # github.com/tikv/pd/tests/server/tso is the integration test of TSO, which will take a long time,
         # will be run independently in the TSO Function Test.
         run: |
-          go list ./... | grep -v -E  "github.com/tikv/pd/server/api|github.com/tikv/pd/tests/client|github.com/tikv/pd/tests/server/tso" > packages.list
```

Comment:
Removing the `tests/client` from here may cause it to run twice in GitHub Actions since we will handle it independently in line 53: https://github.com/tikv/pd/pull/4124/files#diff-b299e538db88efad9dcf6937f6e10a31ba9d0edc04f843b471dcbccaee78da53L53.

Reply:
it will remove all test in tests